tests/test_pipeline/test_pipeline.py

```diff
@@ -757,6 +823,15 @@ def test_backtest_forecasts_sanity(step_ts: TSDataset):
     assert np.all(forecast_df == expected_forecast_df)
 
 
+def test_get_historical_forecasts_sanity(step_ts: TSDataset):
```

Comment:
Could it be `test_sanity_get_historical_forecasts`?

Reply:
We could, but it just follows existing naming (e.g. `test_backtest_forecasts_sanity` above)